neptunecontrib/api/utils.py

```diff
@@ -135,7 +137,8 @@ def extract_project_progress_info(leadearboard, metric_colname, time_colname='fi
     progress_df = _get_daily_experiment_counts(progress_df)
     progress_df = _get_current_best(progress_df)
     progress_df = progress_df[
-        ['id', 'metric', 'metric_best', 'running_time', 'running_time_day', 'experiment_count_day',
+        ['id', 'metric', 'metric_best', 'running_time', 'running_time_day',
```

Comment:
Why are we breking this and other similar lines?

Reply:
some autoformat, fixed.